underfs/hdfs/pom.xml

```diff
@@ -63,9 +63,9 @@
       <id>ufs-hadoop-1</id>
       <dependencies>
         <dependency>
-          <groupId>org.apache.hadoop</groupId>
-          <artifactId>hadoop-core</artifactId>
-          <version>${ufs.hadoop.version}</version>
+          <groupId>org.alluxio</groupId>
+          <artifactId>alluxio-shaded-hdfs</artifactId>
+          <version>${ufs.hadoop.version}-2.0.0-SNAPSHOT</version>
```

Comment:
can we use `${project.version}` for these?

Reply:
removed alluxio version since it does not really depend on alluxio